main.js

```diff
@@ -7,7 +7,11 @@ let win;
 
 function run_program() {
   (async () => {
-    const { stdout } = await execa("client-rpc", ["--network-id", "ab"]);
+    try {
+      const { stdout } = await execa("client-rpc", ["--network-id", "ab"]);
```

Comment:
can `["--network-id", "ab"]` (+ connection strings) be provided externally by the user?

Reply:
yes